snapcraft/internal/errors.py

```diff
@@ -60,3 +60,14 @@ def __init__(self, *, part_name, other_part_name, conflict_files):
         super().__init__(part_name=part_name,
                          other_part_name=other_part_name,
                          file_paths='\n'.join(sorted(spaced_conflict_files)))
+
+
+class MissingPackageError(SnapcraftError):
+
+    fmt = (
+        'A required package is missing, please install these packages:'
```

Comment:
This is a little wrong, because it says:
"A requried package" -> singular
"install these packages:" -> plural

Maybe, we can say: "One or more packages are missing, please install:"